src/actions/rest/cdelete.js

```diff
@@ -0,0 +1,37 @@
+exports.command = 'cdelete';
+exports.describe = 'performs a DELETE request';
+
+exports.builder = (yargs) => yargs
+  .option('url', { describe: 'endpoint to make a DELETE request', demanOption: true, type: 'string' })
+  .example('$0 cdelete --endpoint blah')
+  .example('$0 cdelete blah');
+
+const isEmpty = require('lodash/isEmpty');
+const axios = require('axios');
+const startsWith = require('lodash/startsWith');
+
+const {
+  getConfigValue,
+} = require('../../lib/utils');
+
+exports.handler = async (argv) => {
+  if (!argv || isEmpty(argv.endpoint)) {
+    throw new Error('Parameter --endpoint cant be empty');
+  }
+
```

Comment:
turn all this from line 22 to 30 to a function on `util/lib` therefore preventing duplicated code among rest actions

Reply:
- [ ] TODO